.eslintignore

```diff
@@ -18,8 +18,14 @@ src/styles.js
 src/utils/**
 src/__tests__/**
 
+src/jest.config.json
+src/store.js
+src/styles.js
+src/routes.js
+src/registerServiceWorker.js
+src/index.js
 
-src/components/Admin/**
+# src/components/Admin/**
```

Comment:
Please just remove this line, same as in `.prettierigonore` file

Reply:
Done